### ISSUE TITLE
VmPolicy violation floods logs: IllegalAccessException: Tried to access visual service WindowManager from a non-visual 

### ISSUE THREAD

**Issue by kirillzh** [DataDog/dd-sdk-android#1272]
**Describe what happened**

Logs are flooded when  strict `VmPolicy` is enabled.

**Steps to reproduce the issue:**
1. Configure strict mode in app:
```kotlin
override fun onCreate() {
  StrictMode.setVmPolicy(
    StrictMode.VmPolicy.Builder()
      .detectAll()
      .penaltyLog()
      .build()
  )
```
2. See logs flood with exception:
```
22:30:33.733  E  Tried to access visual service WindowManager from a non-visual Context:FooApplication@bab1093 WindowManager should be accessed from Activity or other visual Context. Use an Activity or a Context created with Context#createWindowContext(int, Bundle), which are adjusted to the configuration and visual bounds of an area on screen.
                 java.lang.IllegalAccessException: Tried to access visual service WindowManager from a non-visual Context:FooApplication@bab1093
                  at android.app.ContextImpl.getSystemService(ContextImpl.java:2115)
                  at android.content.ContextWrapper.getSystemService(ContextWrapper.java:900)
                  at com.datadog.android.rum.internal.vitals.VitalFrameCallback.detectRefreshRateScale(VitalFrameCallback.kt:60)
                  at com.datadog.android.rum.internal.vitals.VitalFrameCallback.doFrame(VitalFrameCallback.kt:33)
                  at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1229)
                  at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
                  at android.view.Choreographer.doCallbacks(Choreographer.java:899)
                  at android.view.Choreographer.doFrame(Choreographer.java:827)
                  at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
                  at android.os.Handler.handleCallback(Handler.java:942)
                  at android.os.Handler.dispatchMessage(Handler.java:99)
                  at android.os.Looper.loopOnce(Looper.java:201)
                  at android.os.Looper.loop(Looper.java:288)
                  at android.app.ActivityThread.main(ActivityThread.java:7872)
                  at java.lang.reflect.Method.invoke(Native Method)
                  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

**Additional context**
 - Android OS version: 33
 - Device Model: emulator
 - Datadog SDK version: 1.17.0

**Comment by 0xnm** [comment]
Hello @kirillzh! Thanks for letting us know about this issue. We will take a look on that, and in the meantime if this is a blocker for you, you can use previous version `1.16.0` (you can find changelogs [here](https://github.com/DataDog/dd-sdk-android/releases)) which shouldn't have this problem.

**Comment by 0xnm** [comment]
We have released version [1.17.1](https://github.com/DataDog/dd-sdk-android/releases/tag/1.17.1) with the revert of the change causing this issue.
